src/Sarif.Viewer.VisualStudio/ErrorListService.cs

```diff
@@ -130,15 +137,24 @@ private void WriteRunLogToErrorList(RunLog runLog)
                         document = location.FullyQualifiedLogicalName;
                     }
 
-                    IRuleDescriptor rule = GetRule(runLog, result.RuleId);
-                    message = result.GetMessageText(rule, concise: true);
+                    rule = GetRule(runLog, result.RuleId);
+                    shortMessage = result.GetMessageText(rule, concise: true);
```

Comment:
Consider an enum `ResultMessageFormat` with values `Short` and `Full` instead of the Boolean.